src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  justify-content: center;
+
+  &__item {
+    width: 16px;
+    height: 16px;
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+}
+
+.stars--4 .stars__item:nth-child(-n + 4) {
+  background-image: url(/src/images/star-active.svg);
```

Comment:
The background-image property uses a relative path to the 'star-active.svg' file. Verify that this path is correct and that the file exists at the specified location to ensure the active star image is displayed correctly.